src/v/cluster/scheduling/partition_allocator.h

```diff
@@ -131,7 +138,10 @@ class partition_allocator {
 
 private:
     std::error_code
-    check_cluster_limits(const allocation_request& request) const;
+    check_cluster_limits(const uint64_t new_partitions_requested) const;
```

Comment:
Can we call this `new_partition_replicas_requested` and add a small comment to the method about this being replication factor * partitions.